src/components/Popover.tsx

```diff
@@ -0,0 +1,65 @@
+import { ReactNode, useState } from "react";
+import styled from "styled-components";
+
+interface PopoverContentProps {
+  title: string;
+  fn: () => void;
+}
+interface PopoverProps {
+  children: ReactNode;
+  content: PopoverContentProps[];
+}
+
+type isVisible = boolean;
+
+function Popover({ children, content }: PopoverProps) {
+  const [visible, setVisible] = useState<isVisible>(false);
```

Comment:
isVisible이라는 타입이 굳이 정의될 필요가 있었는지 모르겠습니다.
아래와 같이 수정하시는게 좋을것 같습니다.
```suggestion
  const [visible, setVisible] = useState<boolean>(false);
```